root/etc/e-smith/events/actions/nethserver-dc-user-modify

```diff
@@ -40,3 +46,35 @@ if ($? != 0) {
     die("[ERROR] User $userName modification failed\n");
 }
 
+
+#Create a temporary ldif file (root:root 0600)
+my $fileName = File::Temp->new(
+    UNLINK   => 0,
+    TEMPLATE => "$userName.XXXXXXXXXX",
+    DIR => '/var/lib/machines/nsdc/var/spool/'
+);
+
+#create the ldif content
+my $contentFile = qq (
+dn: cn=$userName,cn=Users,$baseDN
+changetype: modify
+replace: loginShell
+loginShell: $shell
+);
+
+#print the ldif content in the tempfile
+print $fileName $contentFile;
```

Comment:
Is `$fileName` a file descriptor or the file name (or both)?

After the print a `close()` is required.

Reply:
$fileName is the full path: eg /var/lib/machines/nsdc/var/spool/stephane-xhdghgdh